src/test_c2s_srv.c

```diff
@@ -502,7 +501,7 @@ int test_c2s(Connection *ctl, tcp_stat_agent *agent, TestOptions *testOptions,
 
   activeStreams = connections_to_fd_set(c2s_conns, streamsNum, &rfd, &max_fd);
 
-  while (activeStreams > 0 && (secs() - start_time) < testDuration) {
+  while (activeStreams > 0 && (secs() - start_time - 1) < testDuration) {
```

Comment:
I understand, and will LGTM the one-second extension. Its necessity just makes me a little sad.